packages/forms/src/components/ui/radio/RadioButton.module.css

```diff
@@ -5,7 +5,7 @@
   --swui-radiobutton-animation-time: var(--swui-animation-time-medium);
   /* Icon */
   --swui-radiobutton-icon-color: var(--swui-field-indicator-active-color);
-  --swui-radiobutton-disabled-icon-color: var(--swui-separator-color);
+  --swui-radiobutton-disabled-icon-color: var(--swui-field-text-color-disabled);
```

Comment:
You keep using `--swui-field-text-color-disabled`, but that variable has a specific role. It represents the text color of disabled input fields. If we wan't to change the text color of disabled input fields, then the RadioButton will break.